50_wa_diff_overdose.py

```diff
@@ -0,0 +1,299 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import altair as alt
+from plotnine import *
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+
+# load in pre-cleaned deaths data for Washington and comparison states
+deaths = pd.read_csv("05_cleaned_data/deaths_wa.csv")
```

Comment:
Once you have loaded the data, you should check the observations and attributes in the dataset first. 
1. Use deaths.columns to check all the columns present. 
2. Use head(x),tail(x) or sample(x) function to look at x number of observations.
3. Create a new dataset, that only contains the necessary columns/attributes. This makes computations easier. 
4. Repeat step 2. 

If you follow the steps above, you would notice that certain attributes like Year,Month, Population have float values which is not appropriate and leaves a chance of creating problems. If you follow the steps above, this becomes easier to identify and correct. Since Year and Population attributes have an important part in the analysis, this needs to be addressed.

Reply:
I converted the types of variables you mentioned here in my ipynb, please check them out in the main branch!

Reply:
I did check out the new files. It's good that you have made these changes.